tests/python/contrib/test_hexagon/conftest.py

```diff
@@ -202,3 +202,29 @@ def terminate_rpc_servers():
     yield []
     if serial == "simulator":
         os.system("ps ax | grep tvm_rpc_x86 | awk '{print $1}' | xargs kill")
+
+
+aot_host_target = tvm.testing.parameter(
+    "c",
+    "llvm -keys=hexagon -link-params=0 -mattr=+hvxv68,+hvx-length128b,+hvx-qfloat,-hvx-ieee-fp -mcpu=hexagonv68 -mtriple=hexagon",
+)
+
+
+@pytest.fixture()
```

Comment:
```suggestion
@tvm.testing.fixture
```

and similarly else where

Reply:
done, thanks!